api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,486 @@
+
+            Stream initialization will fail if `stream_limit` is lower than
+            `batch_limit`.
+          type: integer
+          format: int32
+          required: false
+        - name: batch_flush_timeout
+          in: query
+          description: Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached before this time is reached the messages are immediately flushed to the client.
+          type: integer
+          format: int32
+          required: false
+        - name: stream_timeout
+          in: query
+          description: |
+            Maximum time in seconds a stream will live before being interrupted.
+            If unspecified will stream indefinately.
+
+            If this timeout is reached any pending messages (in the sense of
+            `stream_limit`) will be flushed to the client.
+
+            Stream initialization will fail if `stream_timeout` is lower than
+            `batch_flush_timeout`.
+          type: integer
+          format: int32
+          required: false
+        - name: batch_keep_alive_limit
+          in: query
+          description: Maximum number of keep-alive messages to get in a row before closing the connection. Unlimited by default.
+          type: integer
+          format: int32
+          required: false
+      responses:
+        '200':
+          description: |
+            Starts streaming to the client.
+            Stream format is a continuous series of
+            `SimpleStreamEvent`s separated by `\n`
+          schema:
+            $ref: '#/definitions/SimpleStreamEvent'
+        '400':
+          description: Bad syntax
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Not found. Typically in case of requesting from a non existing topic of partition.
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Not found. Typically in case of requesting from a non existing topic of partition.
+          schema:
+            $ref: '#/definitions/Problem'
+    post:
+      description: |
+        Post a one or more `Event`s to a given `Topic`.
+        The partition is implicitly controlled by the partitioning_key attribute of the event. Nakadi guarantees that
+        all events with the same partitioning_key end up in the same partition.
+      parameters:
+        - name: topic
+          in: path
+          description: Topic name where to send events to
+          type: string
+          required: true
+        - name: event
+          in: body
+          description: The body contains an array of events being sent.
+          required: true
+          schema:
+            type: array
+            description: An array of events to be pushed to Nakadi
+            items:
+              $ref: '#/definitions/Event'
```

Comment:
Today I was thinking that maybe it will be a good idea to make it an Array of Array of Events, with inner arrays being kind of atomic batches, and the server would not accept the next batch until the current one is committed into the store.

What do you think guys? Would that make sense, or it is too complicated?